### DISCUSSION TITLE
Default parser from Parcel TS with Jest

### DISCUSSION THREAD

**Discussion by sandhilt** [parcel-bundler/parcel#9356] [Q&A]
How I can share default parser from Parcel v2 with Jest?
It's only possible using babel?

**Comment by achrafelmakaoui** [comment]
Parcel v2 is a JavaScript bundler, and Jest is a testing framework commonly used for JavaScript applications. If you want to use the default parser from Parcel v2 with Jest, you might need to ensure that Jest understands how to handle the file types and features that Parcel supports.*
_Here are some general steps you can take to integrate Parcel v2 with Jest:_

- Make sure you have the required packages installed. You might need to install @babel/core, @babel/preset-env, and other related packages.
- npm install --save-dev @babel/core @babel/preset-env babel-jest

_Configure_ Babel:_
Create a Babel configuration file (e.g., .babelrc or babel.config.js) to specify the preset used by Parcel. This is essential because Jest will use Babel to transpile your code.
// .babelrc or babel.config.js
{
  "presets": [
    ["@babel/preset-env", { "targets": "> 0.25%, not dead" }]
    // Add any other presets or plugins as needed
  ]
}

**Comment by mischnic** [comment]
Not currently, but it would be nice to have, see https://github.com/parcel-bundler/parcel/issues/2396
